implement_strstr().py

```diff
@@ -0,0 +1,6 @@
+class Solution:
+    def strStr(self, haystack: str, needle: str) -> int:
+        if needle in haystack:
+            return(haystack.index(needle))
+        else:
+            return -1
```

Comment:
This is a common thing that can be refactored
```suggestion
        if needle in haystack:
            return haystack.index(needle)
        return -1
```

Reply:
they can give you something like what you did, and ask you: What can be improved? or What is wrong?

Reply:
thanks for the clarification